pkg/api/unversioned/time.go

```diff
@@ -76,6 +76,12 @@ func (t Time) Rfc3339Copy() Time {
 	return Time{copied}
```

Comment:
Let's remove it, since it is unused. It can be added back in if someone needs it.